.travis.yml

```diff
@@ -40,6 +40,8 @@ before_install:
   - if [[ $TEST_COVERAGE != 'true' ]]; then phpenv config-rm xdebug.ini || return 0 ; fi
 
 install:
+  - composer config minimum-stability dev
+  - composer config prefer-stable true
```

Comment:
It cannot work, as composer v2 is not stable... yet.

Still tests does not run with composer v2, but we need run tests with composer v2 so we can make sure the feature  is implemented correctly.

Reply:
composer remove `dealerdirect/phpcodesniffer-composer-installer` on travis build?

Reply:
oh, I know, need update the composer exec as well in travis

Reply:
reverted.